components/Modal/UnlockedStakeTokenModal.tsx

```diff
@@ -0,0 +1,159 @@
+import Button from 'components/Common/Button'
+import InputText from 'components/Common/InputText'
+import Modal from 'components/Common/Modal'
+import IconBack from 'components/Icon/IconBack'
+import { GAS_FEE } from 'constants/gasFee'
+import { useNearProvider } from 'hooks/useNearProvider'
+import { ModalCommonProps } from 'interfaces/modal'
+import { FunctionCallOptions } from 'near-api-js/lib/account'
+import { parseNearAmount } from 'near-api-js/lib/utils/format'
+import React, { useEffect, useState } from 'react'
+import near, { CONTRACT, getAmount } from 'services/near'
+import { parseParasAmount, prettyBalance } from 'utils/common'
+
+interface UnlockedStakeModalProps extends ModalCommonProps {
+	userStaked: string
+	isTopup: boolean
+	lockedBalance: number
+	isWithinDuration: boolean
+	claimableRewards: {
+		[key: string]: string
+	}
+}
+
+const UnlockedStakeTokenModal = (props: UnlockedStakeModalProps) => {
+	const { accountId } = useNearProvider()
+	const [max, setMax] = useState<number>(0)
+	const [inputValue, setInputValue] = useState<string>('')
```

Comment:
`max` perlu dalam state ga? soalnya bisa langsung pake `lockedBalance` kan? kalo ga butuh2 banget mutable, gaperlu state sama effects. recommendationnya gini gan. 
variable `max` gua bikin jadi `maxToUnlock` biar ga bingung

```js
const maxToUnlock = props.lockedBalance / 10 ** 18
```
yang code dibawah ini jadi ga perlu
```js
const [max, setMax] = useState<number>(0)
const setDefaultMax = () => {
  setMax(Number(prettyBalance(`${props.lockedBalance}`, 18)))
}
useEffect(() => {
  if (accountId) {
    setDefaultMax()
  }
}, [])
```

reference
https://beta.reactjs.org/learn/you-might-not-need-an-effect#updating-state-based-on-props-or-state

Reply:
oiya gan kalo di unlocked ga perlu mutable max nya, thank you gan